Makefile

```diff
@@ -104,10 +104,14 @@ third-party-try-gmp: FORCE
 	fi
 
 third-party-test-venv: FORCE
-	cd third-party && $(MAKE) test-venv
+	-@if [ -z "$$CHPL_TEST_VENV_DIR" ]; then \
```

Comment:
@ronawho / @ben-albrecht : Why do we use a `@` here? (which causes failures to be ignored)  In nightly testing, this seems to sweep real problems under the rug temporarily until we actually try to run the test system later (e.g., see issue #395).  If there's no good reason for it, I'm inclined to remove it.

Reply:
I'm not sure, and I don't remember Makefile changes well enough to have an answer. Usually I'm copying from some other makefile. Feel free to change it if you think it'd be valuable.

Reply:
Oops, I got my Makefile symbols backwards.  The `-` ignores errors, the `@` squashes printing the command to stdout.

Reply:
Opened PR #13698 to address this.